TemplateManagement/PowerShell/ActiveDirectoryJoinV2/Join-Domain-StudentVM.ps1

```diff
@@ -56,9 +56,9 @@ try {
     New-Item -Path $logFile -ItemType File
 
     # Check Windows 10 / 11 Operating system
-    if (!([System.Environment]::OSVersion.Version.Major -match "10" -or [System.Environment]::OSVersion.Version.Major -match "11")) {
+    if ([System.Environment]::OSVersion.Version.Major -lt "10") {
```

Comment:
We should probably also confirm windows correct?  (since someone could try running the script on Linux via powershell core and if the OS version happens to line up here, like Ubuntu 22.04?)